Source/Softswitch/src/softswitch_common.cpp

```diff
@@ -523,48 +532,62 @@ void softswitch_onReceive(ThreadCtxt_t* ThreadContext, volatile void* recv_buf)
                 ThreadContext->pendCycles = 1;
                 break;
 #endif
-            default:            // Unused reserved Opcode - log it.
-                                handler_log(5, "BAD-OP %d", opcode);
-
+            default:    // Unused reserved Opcode - log it.
+                        //handler_log(5, "BAD-OP %d", opcode);
+                        break;
```

Comment:
But why is this commented out? It looks useful...

Reply:
Yeah, I need to come back and address these. The "tweaked" handler log requires a `deviceInstance` in context to get the source address. We don't have a 'deviceInstance' in context at this point, so the macro fails. 

I could use the raw __handler_log, with a fabricated source, but we then won't know the source.

I guess I could just pretend to be the first device instance in this case (and the previous one)

Reply:
That's a knotty one - could you reserve a special device ID for this purpose?

Reply:
That is a possibility, the maximum value?

Reply:
> Will grab the index of the first device on the Softswitch so that we get the correct hardware address. Add comment to source.

Addressed in e1b5924dde536e02ebf62243cd62ee007173e730